src/controller/CHIPDeviceController.cpp

```diff
@@ -704,18 +706,22 @@ CHIP_ERROR DeviceCommissioner::EstablishPASEConnection(NodeId remoteDeviceId, Re
 #if CONFIG_NETWORK_LAYER_BLE
     if (params.GetPeerAddress().GetTransportType() == Transport::Type::kBle)
     {
+        MATTER_LOG_METRIC_SCOPE(chip::Tracing::kMetricPASESessionBLE);
```

Comment:
why is this LOG_METRIC_SCOPE instead of the existing  MATTER_TRACE_SCOPE ?

We are forking and not re-using. This needs changing. We can even drop the old one if we think "group + label" was a wrong approach and we always want a single key. 

However:
   - this scope is NOT a metric (there is no metric value to speak of. sending 0 because it is convenient is wrong)
   - we should not fork the same thing once with one key and once with two keys.

Reply:
This is just a sample code to show usage.  I will be reverting all the changes in this file as mention in our conversation earlier.

Reply:
Please see: https://github.com/project-chip/connectedhomeip/pull/32223/files#r1500369705. The existing TRACE macros are limiting.